In [1]:
from astropy import units as u
from astroquery.atomic import AtomicLineList

# Look at the supposed Ca I lines

## The 7890 line

This looks a lot like the [Fe III] lines

In [17]:
wavelength_range = (7890 * u.Angstrom, 0.2 * u.Angstrom)
#wavelength_range = (6500 * u.Angstrom, 6600 * u.Angstrom)
#AtomicLineList.FORM_URL = "https://www.pa.uky.edu/~peter/newpage/"
res = AtomicLineList.query_object(
    wavelength_range=wavelength_range,
    wavelength_type='Air',
    minimal_abundance="6",
    depl_factor="0",
    #element_spectrum='H I',
    #get_query_payload=True,
)

In [18]:
res

<Table length=18>
LAMBDA AIR ANG SPECTRUM  TT  ...   J J     A_ki       LEVEL ENERGY  CM 1   
     str8        str8   str2 ...   str7    str8             str25          
-------------- -------- ---- ... ------- -------- -------------------------
         7608.  Si VII]   E1 ...     1-0     None  1491287.00 -  1504430.00
         7698.    Ar X]   E1 ... 1/2-3/2     None  2555193.00 -  2568152.00
         7788.    Si XI   E1 ...     2-2 1.13E+06  3187600.00 -  3200430.00
         7838.    Si XI   E1 ...     2-3 3.26E+04  3188940.00 -  3201700.00
         7868.    Si XI   E1 ...     2-2 3.61E+05  3188940.00 -  3201650.00
         7873.     N VI   E1 ...     *-* 2.63E+07  4390976.00 -  4403673.70
         7878.    Si XI   E1 ...     2-1 1.59E+05  3200430.00 -  3213120.00
         7888.  Fe III]   E1 ...     2-2     None   167002.00 -   179676.89
        7889.9    N II]   E1 ...     1-2     None   224286.63 -   236957.50
        7889.9 [Ni III]   M1 ...     3-2     None     1360.70 -    14031.60
       7890.07    Ca I]   E1 ...     2-2     None    36575.12 -    49245.80
      7890.154     Ni I   E1 ...     2-1     None    31441.63 -    44112.17
        7890.4     Ar I   E1 ...     3-2     None   113716.55 -   126386.70
         7891.   Al XII   E1 ...     1-2 5.08E+06 15816791.00 - 15829460.00
         7893.      O V   E1 ...     2-3 2.43E+07   818668.00 -   831334.00
         7894.    C III   E1 ...     1-0     None   363613.00 -   376278.10
         7898.   Ca XIX   E1 ...     1-2 5.07E+06 39761380.00 - 39774040.00
        7918.?   Na VII   E1 ...   5/2-* 4.08E+04   1377820.00 -  1390450.?

So it looks like the [Ni III] 7889.9 line is a very good candidate.  It is a nebular forbidden line, so would be expected to be strong. 